modules/appserver-utils/src/main/java/org/wso2/appserver/exceptions/ConfigurationException.java

```diff
@@ -14,21 +14,23 @@
  * KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations
  * under the License.
- *
  */
 
-package org.wso2.appserver.webapp.loader.exceptions;
+package org.wso2.appserver.exceptions;
 
 /**
- * Thrown when the application server encountered an error.
+ * Thrown when the application server failed to build the configurations.
+ *
+ * @since 6.0.0
  */
-public class ApplicationServerException extends Exception {
+public class ConfigurationException extends ApplicationServerException {
+    private static final long serialVersionUID = -3620848532234792303L;
 
-    public ApplicationServerException(String message, Throwable cause) {
+    public ConfigurationException(String message, Throwable cause) {
```

Comment:
I think last we discussed to chenge this to ApplicationServerConfigurationException